code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -98,6 +98,13 @@
 /turf/open/proc/update_visuals()
 	var/list/new_overlay_types = tile_graphic()
 
+	#if DM_VERSION >= 513
+	#warning 512 is stable now for sure, remove the old code
+	#endif
+	
+	#if DM_VERSION >= 512
+	vis_contents = new_overlay_types
```

Comment:
Please do not do this.

Do not assume that only gas overlays will go into vis_contents. 

The reason we use the messy overlay code below is to avoid stepping on any toes of anything else that wants to add overlays to turfs.

Reply:
true, I forgot to do any future proofing after testing it worked